build/ci/evergreen.yml

```diff
@@ -952,6 +952,25 @@ tasks:
           MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
           MCLI_SERVICE: cloud
           E2E_TAGS: atlas,livemigrations
+  # Streams commands tests
+  - name: atlas_streams
+    tags: ["e2e","generic","atlas"]
```

Comment:
```suggestion
    tags: ["e2e","streams","atlas", "assigned_to_jira_team_cloudp_atlas_streams"]
```

Reply:
Resolved in [50a2a7e](https://github.com/mongodb/mongodb-atlas-cli/pull/2138/commits/50a2a7e4e54c1858343d0ccba569af469b7876e1)